docs/configuration/logging.md

```diff
@@ -35,8 +35,10 @@ To collect logs from your Codacy installation using fluentd change `fluentdopera
 
 The fluentd daemonset will send the logs to minio which is also installed by this chart.
 
-In order to send them to our support in case of problems, run the following command locally (replacing the `<namespace>` with the namespace in which Codacy was installed):
+To extract the logs and send them to Codacy's support team in case of problems, you can run the following command locally (replacing the `<namespace>` with the namespace in which Codacy was installed):
 
 ```bash
-kubectl cp <namespace>/$(kubectl get pods -n <namespace> -l app=minio -o jsonpath='{.items[*].metadata.name}'):/export/logs ./logs
+curl https://codacy.github.io/chart/configuration/extract-codacy-logs.sh > extract-codacy-logs.sh && chmod +x extract-codacy-logs.sh && extract-codacy-logs.sh -n <namespace>
```

Comment:
```suggestion
curl -fsSL https://codacy.github.io/chart/configuration/extract-codacy-logs.sh | bash -s -- -n <namespace>
```

Reply:
We could use a shorter version here. Since the other is a single command this seems more concise.